addons/account/wizard/base_document_layout.py

```diff
@@ -31,14 +31,14 @@ def _get_preview_template(self):
 
     def _get_render_information(self, styles):
         res = super()._get_render_information(styles)
-        if (
-            self.env.context.get('active_model') == 'account.move'
-            and self.env.context.get('active_id')
-        ):
-            res.update({
-                'o': self.env['account.move'].browse(self.env.context.get('active_id')),
-                'qr_code': self.qr_code,
-            })
+        data = {
```

Comment:
The `data` variable seems unnecessary. We can just update `res` directly in both cases.
```suggestion
        res.update({
```